src/picongpu/include/particles/ionization/byField/BSIHydrogenLike/AlgorithmBSIHydrogenLike.hpp

```diff
@@ -64,13 +65,16 @@ namespace ionization
         operator()( const BType bField, const EType eField, ParticleType& parentIon )
         {
 
-            const float_X protonNumber = GetAtomicNumbers<ParticleType>::type::numberOfProtons;
-            float_X chargeState = attribute::getChargeState(parentIon);
-
-            uint32_t cs = math::float2int_rd(chargeState);
+            const float_X protonNumber  = GetAtomicNumbers<ParticleType>::type::numberOfProtons;
+            float_X chargeState         = attribute::getChargeState(parentIon);
+            uint32_t cs                 = math::float2int_rd(chargeState);
+            /* ionization potential in atomic units */
+            const float_X iEnergy       = GetIonizationEnergies<ParticleType>::type()[cs];
```

Comment:
Argh! I should move the condition 
```C++
    if (chargeState < protonNumber)
```
before the definition of the ionization energy! Also the `GetIonizationEnergies` trait should receive a fallback for this!